utils/v3/version/get_version.sh

```
@@ -0,0 +1,16 @@
+# Get application version information (public endpoint, no auth required)
+# Usage: ./get_version.sh
+# API_URL=http://localhost:5001 ./get_version.sh
+# API_URL=https://api.lfcla.dev.platform.linuxfoundation.org ./get_version.sh
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# Handle API URL
+. ${SCRIPT_DIR}/../shared/handle_api_url.sh
+
+# Set up curl execution
+API="${API_URL}/v3/ops/version"
```

Shellcheck (0.11.0)

[warning] API appears unused. Verify use (or export if used externally).

(SC2034)